.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: go
 go:
-- "1.12"
+- "1.17"
```

Comment:
Why not upgrade it to 1.19?

Reply:
n a first moment I bumped it into 1.17 version just to make sure that everything was working fine. Just updated it to 1.19!
Thanks, thiagomachado! 😄